charts/incubator/alfresco-infrastructure/templates/pvc.yaml

```diff
@@ -1,34 +1,34 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
-{{- if eq .Values.persistence.volumeEnv "aws" -}}
+{{- if and .Values.persistence.efs.enabled (not .Values.persistence.existingClaim) -}}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs
+provisioner: alfresco.com/efs
+---
 apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: alfresco-volume
 spec:
   capacity:
-    storage: 10Gi
+    storage: {{ .Values.persistence.baseSize | quote }}
   accessModes:
     - ReadWriteMany
+  persistentVolumeReclaimPolicy: {{ .Values.persistence.reclaimPolicy | default "Delete" }}
```

Comment:
We should add a note to the README on the implications of this and point to K8s docs section on it for other values.

Reply:
Yes, will do asap.